app/Http/Controllers/Users/BulkUsersController.php

```diff
@@ -374,7 +374,7 @@ private function logConsumablesCheckin(Collection $consumableUserRows): void
             $logAction->item_type = Consumable::class;
             $logAction->target_id = $consumableUserRow->assigned_to;
             $logAction->target_type = User::class;
-            $logAction->created_at = auth()->id();
+            $logAction->created_by = auth()->id();
```

Comment:
We don't check in consumables though? Should we pull this method altogether?

Reply:
Looks like this was carried over in #15221 from previous behavior but yeah...we don't actually check in consumables. 

Do we want to purge the action log of existing consumable checkins? If so I think we should do another PR that includes removing this and the migration to do the purge...

I'm ok if you'd rather me just pull it from here too.

Reply:
For reference: #16494